doc/2/plugins/plugin-context/accessors/trigger/index.md

```diff
@@ -6,12 +6,15 @@ title: trigger
 
 # trigger
 
-
-
 Triggers a custom event.
 
 This allows interactions with other plugins using [hooks](/core/2/plugins/guides/hooks) or [pipes](/core/2/plugins/guides/pipes).
 
+::: info
+If the event is listened by pipes, the result of the pipe chain will be returned in a promise.
```

Comment:
The result of that call if no pipe is listening to an event should be documented (and it should probably resolve to an undefined value)

Reply:
I'm not sure about this one because pipes are meant to take a variable and may or not apply transformation to it and returning it.  
So IMHO it's normal if the pipe return the variable if there is no listener.

I made this to be able to have some kind of communication between plugins but with the new loosely coupling system I think `ask` event will be more appropriated to do some kind of RPC between plugins.   
And those events will return `undefined` if there is no listener right?

Reply:
Ah, you're right, it makes much more sense to return the first argument untouched if there is no pipe listening. :+1: